server/container_create_linux.go

```diff
@@ -796,13 +694,11 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 	sort.Sort(orderedMounts(mounts))
 
 	for _, m := range mounts {
-		mnt = rspec.Mount{
+		ctr.SpecAddMount(rspec.Mount{
```

Comment:
Maybe pass `rspec.Mount` via a reference, otherwise it's too much data being copied over

Reply:
fixed !